<div class="imageContainer" style="" >
    <img src="img/paintomics_150x690.png" height="50" title="Paintomics LOGO." style=" height: 80px !important; margin-bottom: 20px; "/>
</div>

**Integrative visualization of multiple *omic* data**

For any question on PaintOmics, users can send a mail to [paintomics@cipf.es](mailto:paintomics@cipf.es).

**Welcome to PaintOmics v3.0 documentation.**

**PaintOmics** is a web tool for the integrative visualization of multiple omic datasets onto KEGG pathways.

PaintOmics consists of three simple steps:

* **Data uploading**: typically data matrices, containing, for example, gene expression, metabolite levels and protein levels for the same set of samples.
* **Identifier and Name Matching and Metabolite assignment**: as PaintOmics requires Entrez IDs for working with KEGG pathways, the tool will try to convert the names and identifiers from different sources and databases for the input data. Additionally, it's necessary to specify of metabolite assignments in ambiguity cases.
* **Pathway analysis:** Paintomics 3 displays the relative abundance of omic features on the KEGG pathway diagram. The tools covers a complete analysis workflow, including pathway enrichment and network analysis, heatmaps, trend charts, etc. KEGG pathways can be interactively explored to visualize not only metabolite and gene expression across experimental conditions but also changes in their regulatory elements such as miRNAs, transcription factors, methylation sites, ChIP-seq peaks, etc.

Currently PaintOmics supports integrated visualization of multiple species of different biological kingdoms and offers user the possibility to request any other organism present in the KEGG database.
